### PR TITLE
Newsletter-253: translate into Chinese

### PR DESCRIPTION
**Pull request by hulatown**

### CODE REVIEW

File: _posts/zh/newsletters/2023-05-31-newsletter .md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+- [BTCPay Server 97e7e][]开始为[payjoin][topic payjoin]支付设置[BIP78][]`minfeerate`(最小费率)参数。另请参阅导致此提交的[错误报告][btcpay server #4689]。{% assign timestamp="1:41:56" %}
+
+- [BIPs #1446][]对比特币相关协议的[schnorr 签名][topic schnorr signatures]的[BIP340][]规范进行了小改动和一些补充。此更改允许对消息进行任意长度的签名；以前版本的 BIP 要求消息正好为 32 个字节。描述对 Libsecp256k1 库的相关更改请见 [周报 #157][news157 libsecp]。此更改对共识应用程序中 BIP340 的使用没有影响，因为与[taproot][topic taproot]和[tapscript][topic tapscript] (分别为，BIPs[341][bip341] and [342][bip342])一起使用的签名使用 32 字节消息。
+
+    新增内容描述了如何有效地使用任意长度的消息，推荐了如何使用散列标记前缀，并提供了在不同域中使用相同的密钥（例如签署交易或签署纯文本消息）时提高安全性的建议。 {% assign timestamp="1:43:15" %}
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="27469,27626,25796,2668,2666,4689,1446" %}
+[policy series]: /en/blog/waiting-for-confirmation/
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2023-05-31-newsletter .md:111]: @hulatown would you mind creating a zh version of the policy series similar to the fr version here: https://github.com/bitcoinops/bitcoinops.github.io/pull/1147/files#diff-d5483c9b332b1fe2721efc5852c20afac39aaa0dfcbd390d61d346d296706149

Comment (hulatown) [reply _posts/zh/newsletters/2023-05-31-newsletter .md:111]: Added! Sorry for late reply

Comment (bitschmidty) [inline _posts/zh/newsletters/2023-05-31-newsletter .md:111]:
```suggestion
[policy series]: /zh/blog/waiting-for-confirmation/
```

**Review by bitschmidty — COMMENTED** [review]
Ive opened a new PR (https://github.com/bitcoinops/bitcoinops.github.io/pull/1167) to resolve the merge conflicts and rebased on master and:

- renamed the '_posts/zh/newsletters/2023-05-31-newsletter .md' file to remove the extra space
- changed the '/en/blog/waiting-for-confirmation/' reference to the new /zh/ version

@hulatown can you review https://github.com/bitcoinops/bitcoinops.github.io/pull/1167

File: _posts/zh/newsletters/2023-05-31-newsletter .md (hunk 1 above)
Comment (hulatown) [reply _posts/zh/newsletters/2023-05-31-newsletter .md:111]: Thanks Mike! #1167 looks good to me. Shall I close this PR?

**Pull request closed without merging**